api/docs/developer-system-initialization.md

```diff
@@ -97,10 +97,18 @@ git secrets --scan /path/to/file (/other/path/to/file *)
 git secrets --scan -r /path/to/directory
 ```
 
+## envsubst
+
+`envsubst` is part of the `gettext` package.
+If you are on a mac, run:
+```Shell
+  brew install gettext
+```
+
 ## Initial Smoke Test
 Before doing any development, you must run the following from `/api`:
 ```Shell
-./gradlew compileGeneratedJava appengineRun
```

Comment:
This no longer works (at least for me)
```
$ ./gradlew appEngineRun

> Task :generate_appengine_web_xml FAILED
libproject/generate_appengine_web_xml.rb:6:in `gsub!': no implicit conversion of nil into String (TypeError)
        from libproject/generate_appengine_web_xml.rb:6:in `<main>'
```

Reply:
my understanding is that this is no longer useful since we're all using dev-up now.  If that's not the case, let's file a story to fix it

Reply:
Yep, I'm not sure why that was here. I don't think it was actually ever true... AFAIK